common/api/core-backend.api.md

```diff
@@ -1393,12 +1393,12 @@ export class DocumentPartition extends InformationPartitionElement {
     static get className(): string;
 }
 
-// @beta
+// @public
```

Comment:
As a result of marking backendhubaccess public, also had to mark downloadchangesetarg, downloadchangesetrangearg, and downloadprogress arg public. @GytisCepk any problem with that? We could potentially explicitly mark downloadChangesets and downloadChangeset as @beta if so.

Reply:
wait, i thought the plan was to mark it @internal?

Reply:
Yeah, I started off the PR that way.. but I just don't see how we can mark it internal. imodels-* packages use it, and we expect backends to provide an implementation of BackendHubAccess, which is pretty necessary to do anything useful (open an iModel for example). Additionally marking it internal, means we need to suggest alternatives to some of these docs https://www.itwinjs.org/learning/imodelhub/imodels/createimodel/

Reply:
you should edit the title of this PR

Reply:
Done, thanks

Reply:
I'm not sure whether create/delete iModel are really valid except for tests. Did you ask the iModelHub team?

Reply:
I didn't ask them, but I did get some comfort from the fact that markdown file has this line "To start working with iModelHub, an iModel for an iTwin has to be created. End users should usually create the iModel for an iTwin through iModelHub website." before mentioning createNewIModel.

Reply:
Pushed a couple of small changes per suggestion from hub team.

'create the iModel for an iTwin through iModel manager' and linking to the CreateiModel learning page in the permissions page.